### DISCUSSION TITLE
How do I reload the timeline if a scrub event or screen resize event occurs?

### DISCUSSION THREAD

**Discussion by jaradc** [videojs/video.js#7594] [Q&A] [answered]
I'm using AWS MediaConvert and outputting several different resolutions (ex: 1920 x 1080, 1280 x 720, ... 426 x 240).
Problem is, when I either:
 1. scrub the timeline
 2. or resize the screen

sometimes the playback is using a lower-quality resolution, so the video looks blurry temporarily until about 1 minute later.

My question is, the video seems to load future parts, which is what I want. But if I scrub the timeline or resize the screen, I want it to update with the best looking resolution for that size.

**How do I reload the timeline if a scrub event or screen resize event occurs?**

By "reload" I mean I want to "reset" whatever parts of the timeline were downloaded and redownload them as a user. I don't want a user (myself) to have to wait 1 minute for the resolution to recognize.

**Can I change how far into the future to download?** Ex: 10 seconds?

![image](https://user-images.githubusercontent.com/12854767/148305063-12a09769-da8d-4636-b034-34260d59aae4.png)

**Comment by jaradc** [comment]
Can someone at least inform me what Video.js calls the part of the video that loads in-advance on the timeline? Is that preload? load? media loader? Range something?

**Comment by mister-ben** [comment, marked as answer]
It's the  "forward buffer". It's possible to tweak it at `videojs.Vhs.GOAL_BUFFER_LENGTH`, but it's not recommended.
